react-frontend/public/configEnv.js

```diff
@@ -1,6 +1,8 @@
 // Default local env variables
 // Use in place of the REACT_APP style env variables.
 
+window.PILOT_MODE = false;
```

Comment:
This is the flag that switches between observer and pilot modes. Could also be added to `configEnv-production.js`.